simple_template_server/README.md

```diff
@@ -0,0 +1,50 @@
+Template Development Server
+===================================
+
+This is a simple Snippet Server for development of Snippet Templates. It ignores
+the client configuration including locale, Firefox version and release channels
+and just returns all the snippets to the requesting client. If you only care for
+Snippet Template development this is the server for you.
+
+Installation
+------------
+
+1. Create and activate a Python virtualenv
+
+    ``` shell
+    virtualenv venv
+    . venv/bin/activate
+    ```
+
+2. Install the needed packages
+
+    ``` shell
+    pip install -r requirements.txt
+    ```
+
+Using the server
+----------------
+
+1. Run the server
+
+    ``` shell
+    python webserver.py
+    ```
+
+    The server listens on *http://0.0.0.0:8000*
```

Comment:
Cool, it works without the addon. Which is my preferred way.

I update the about:config value. I close and open my browser. I then open about:home in a private tab.

I think we can just clear up some of the docs though as a lot of this was only obvious after chatting with you.